Multisig/Info.plist

```diff
@@ -58,6 +58,8 @@
 	<string>Safe need to use camera to scan QR code.</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>This will allow Safe Multisig to access your Face ID to open app and sign transactions.</string>
+	<key>NSUserTrackingUsageDescription</key>
+	<string></string>
```

Comment:
The actual description string is missing. I think we should take the string from our terms screen, but it's better to check wording with @tschubotz:

```Anonymized app usage data will be used to improve the app```

Reply:
Where will this be displayed?

Reply:
In the alert (https://github.com/gnosis/safe-ios/pull/1141#issue-630607161) between the text and button on that screenshot

Reply:
https://developer.apple.com/app-store/user-privacy-and-data-use/
I made it empty on purpose. So it will only display the bold part

Reply:
@sche on that site, it clearly says:

```You must also include a purpose string in the system prompt that explains why you’d like to track the user.```

Reply:
And more:

```Use the purpose string to explain what this data will be used for to help the user understand what they’re opting in to share. ```

Reply:
ah ok, I think 


> `Anonymized app usage data will be used to improve the app.`

sounds good then!